marshaling.go

```diff
@@ -233,6 +237,15 @@ func convert(ctx convertCtx, value Value) (reflect.Value, error) {
 		return locus, nil
 	}
 
+	// if we have a type that uses a custom converter, use it instead of standard conversion
```

Comment:
One possible issue of moving this here is that if you have a custom converter for a worksheet, or a slice, it needs to take precedence, otherwise we're breaking the contract. Did you check this?

Reply:
checking, will add to tests